docs/root/configuration/http/http_filters/adaptive_concurrency_filter.rst

```diff
@@ -64,9 +64,11 @@ The gradient is calculated using summarized sampled request latencies (sampleRTT
 
 .. math::
 
-    gradient = \frac{minRTT}{sampleRTT}
+    gradient = \frac{minRTT + buffer}{sampleRTT}
```

Comment:
Can you add more of an explanation in the main docs on how the buffer is calculated and what it is for? Also a ref link to the configuration field would be useful.